monkey/agent_plugins/exploiters/rdp/manifest.yaml

```diff
@@ -0,0 +1,17 @@
+name: RDP
+plugin_type: Exploiter
+supported_operating_systems:
+  - linux
+  - windows
+target_operating_systems:
+  - windows
+title: RDP Exploiter
+version: 1.0.0
+description: Attempts a brute-force attack over RDP using known credentials.
+safe: true
+remediation_suggestion: >-
+    Change user passwords to complex one-use passwords that are not shared with other computers on the network.
+
+    The machine is vulnerable to an RDP attack.
```

Comment:
This sounds a little dramatic. We were able to login via a remote access protocol. We had correct credentials.

Reply:
Yeah. I dropped [T1210](https://attack.mitre.org/techniques/T1210/) from the tags for this same reason -- we're not _exploiting_ the protocol, just using it.